src/main/java/domain/RacingCar.java

```diff
@@ -3,21 +3,39 @@
 import movingStrategy.Moveable;
 
 public class RacingCar {
-    private final Moveable moveable;
-    private Integer position;
+    private final String name;
+    private final Position position;
 
-    public RacingCar(Moveable moveable) {
-        this.position = 0;
-        this.moveable = moveable;
+    public RacingCar(String name, Integer initialPosition) {
+        this.name = name;
+        this.position = new Position(initialPosition);
     }
 
-    public Integer whereIsCar() {
-        return this.position;
+    public void move(Moveable moveable) {
+        if (moveable.isMoveable()) {
+            this.position.add(1);
```

Comment:
```suggestion
            this.position.add(MOVE_VALUE);
```

다른 동료 개발자가 `1`개념을 쉽게 이해하고, 유지보수시 찾기 쉽게하려면 어떻게 해야할까요?